apps/website/src/components/forms/EntryForm.tsx

```diff
@@ -219,7 +219,9 @@ export const EntryForm: React.FC<{
               />
               {config.requireShippingAddress && <ShippingAddressFieldset />}
               {config.hasRules && <EntryRulesFieldset form={form} />}
-              <ConsentFieldset />
+              <ConsentFieldset
+                withShippingAddress={config.requireShippingAddress}
+              />
```

Comment:
I would think it makes more sense to have the marketing consent be part of the consent section, rather than part of the section where you provide the details?

Reply:
Sure, i don't have strong feelings, just felt very checkbox heavy down there plus it might be more clear what the email address entered is used for if the consent is right there?!

Reply:
In some ways I agree it's more apparent, but at the same time, I think I'd expect to see checkboxes relating to data processing being listed under data processing.

Reply:
With moving it to the bottom, it might also make sense to word it more like:

```
Additionally, I am happy for Alveus Sancatury Inc. to use my email for direct marketing relating to the Sanctuary, including from its partners, such as for news, fundraisers, and other promotions.
```

Reply:
Okay that sounds good. Will do. We should probably still mention the partners since I understand Alveus wants to share the data with the merch company.

Reply:
Yah, I kept the partners mention in there

Reply:
Oh im dumb. So hard to read in the scroll box :)